libnopegl/src/api.c

```diff
@@ -196,11 +198,20 @@ void ngli_ctx_reset(struct ngl_ctx *s, int action)
 #if defined(TARGET_ANDROID)
     ngli_android_ctx_reset(&s->android_ctx);
 #endif
+    ngli_hmap_freep(&s->text_builtin_atlasses);
     ngli_pgcache_reset(&s->pgcache);
     ngli_gpu_ctx_freep(&s->gpu_ctx);
     ngli_config_reset(&s->config);
 }
 
+void ngli_free_text_builtin_atlas(void *user_arg, void *data)
+{
+    struct text_builtin_atlas *atlas = data;
+    ngli_distmap_freep(&atlas->distmap);
+    memset(atlas->char_map, 0, sizeof(atlas->char_map));
```

Comment:
Do we really need to memset char_map here since atlas is destroyed anyway ?

Reply:
No but I'm just being paranoid about dangling references (a `memset` of 256 bytes should not be expensive, especially at this stage)

Reply:
I'm asking because we don't do that in other part of the codebase for allocated structure.

Reply:
We usually do it for pointers, but yeah sure I can get rid of it if you think it's not worth it

Reply:
I think you can drop it, unless we decide to do that everywhere.

Reply:
Alright, removed